exercises/atbash-cipher/canonical-data.json

```diff
@@ -1,6 +1,6 @@
 {
   "exercise": "atbash-cipher",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Should be 1.2.0 since you are adding tests.

Reply:
Okay, thanks I didn't know what the protocol was.